src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -485,3 +485,31 @@ public static DisplayInformation GetForMonitor(IntPtr monitor)
     }
 #endif
 }
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+#if UAC_VERSION_1
+#if NET
+    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.10240.0")]
+#endif
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class DataTransferManagerInterop
+    {
+        private static IDataTransferManagerInterop dataTransferManagerInterop = DataTransferManager.As<IDataTransferManagerInterop>();
+
+        public static DataTransferManager GetForWindow(IntPtr appWindow)
+        {
+            Guid iid = GuidGenerator.CreateIID(typeof(IDataTransferManager));
+            return (DataTransferManager)dataTransferManagerInterop.GetForWindow(appWindow, iid);
+        }
+
+        public static void ShowShareUIForWindow(IntPtr appWindow)
+        {
+            dataTransferManagerInterop.ShowShareUIForWindow(appWindow);
+        }
+    }
+}

```

Comment:
missing `#endif` at end for UAC_VERSION_1 is causing CI failure.

  ComInteropHelpers.cs(525,1): error CS1027: #endif directive expected

Reply:
![image](https://github.com/microsoft/CsWinRT/assets/49179966/e900b588-68d7-428b-997f-71ed0f4f7caf)
Has updated just now